src/applications/SimulationSpecification.java

```diff
@@ -137,16 +137,16 @@ private void activateJobs(MachineShopSimulator machineShopSimulator) {
 	private void setupJob(MachineShopSimulator machineShopSimulator, int p) {
 		machineShopSimulator.machine[p].setActiveJob((Job) machineShopSimulator.machine[p].getJobQ()
 		        .remove());
-		machineShopSimulator.machine[p].setTotalWait(machineShopSimulator.machine[p].getTotalWait() + machineShopSimulator.timeNow
+		machineShopSimulator.machine[p].setTotalWait(machineShopSimulator.machine[p].getTotalWait() + machineShopSimulator.getTimeNow()
 		        - machineShopSimulator.machine[p].getActiveJob().getArrivalTime());
 		machineShopSimulator.machine[p].setNumTasks(machineShopSimulator.machine[p].getNumTasks() + 1);
 		int t = machineShopSimulator.machine[p].getActiveJob().removeNextTask();
-		machineShopSimulator.eList.setFinishTime(p, machineShopSimulator.timeNow + t);
+		machineShopSimulator.eList.setFinishTime(p, machineShopSimulator.getTimeNow() + t);
 	}
 
 	public  SimulationResults runSimulation(MachineShopSimulator machineShopSimulator) {
 	    machineShopSimulator.largeTime = Integer.MAX_VALUE;
-	    machineShopSimulator.timeNow = 0;
+	    machineShopSimulator.setTimenow(0);;
```

Comment:
It looks like setTimenow is only ever passed in a value of 0. I think this shows us that it might be a good idea to make a setter function that takes no parameter.